www/templates/main-recommendations.html

```diff
@@ -34,6 +35,15 @@ <h3>Reviews</h3>
          <button class="button button-block button-balanced button-outline back" ng-click="clickX()">Back</button>
      </div>
    </div>
+   <div id="tripSelection" class="modal">
+     <div class="modal-content">
+       <div align="left">
+       <h3>Past Cleaned Trips</h3>
+       Cleaned trips will be pulled and shown here. 
```

Comment:
is there a reason why this can't be integrated directly with the diary? Is it the distinction of cleaned versus draft trips or ???

Reply:
because it looks like you already have a button in the detail